Code/Framework/AzToolsFramework/AzToolsFramework/UI/DocumentPropertyEditor/DocumentPropertyEditor.h

```diff
@@ -16,39 +16,56 @@
 #include <AzToolsFramework/UI/DocumentPropertyEditor/PropertyHandlerWidget.h>
 
 #include <QHBoxLayout>
-#include <QPointer>
-#include <QFrame>
+#include <QScrollArea>
 
 #endif // Q_MOC_RUN
 
+class QCheckBox;
+
 namespace AzToolsFramework
 {
     class DocumentPropertyEditor;
 
     class DPELayout : public QHBoxLayout
     {
+        Q_OBJECT
+
+        signals:
+        void expanderChanged(bool expanded);
+
         // todo: look into caching and QLayoutItem::invalidate()
     public:
         DPELayout(int depth, QWidget* parentWidget = nullptr);
+        virtual ~DPELayout();
+        void setExpanderShown(bool shouldShow);
+        void setExpanded(bool expanded);
+        bool IsExpanded();
```

Comment:
repeated nit: these and the methods below should be UpperCamelCase

Reply:
fixed, thanks!